lib/octopus/proxy_config.rb

```diff
@@ -78,6 +77,7 @@ def current_group=(group_symbol)
     end
 
     def current_slave_group
+      return nil if Thread.current[CURRENT_SHARD_KEY] == :master
```

Comment:
would this trigger a problem when assignment is attempted at https://github.com/change/octopus/blob/49b56cfeec24889899763ca3ac83e41e7577c3b5/lib/octopus/proxy_config.rb#L34 ?

Reply:
ahh, assignment is handled in L84, so probably not